pandas/tools/tile.py

```diff
@@ -191,7 +197,11 @@ def _bins_to_cuts(x, bins, right=True, labels=None, retbins=False,
     ids = bins.searchsorted(x, side=side)
 
     if len(algos.unique(bins)) < len(bins):
-        raise ValueError('Bin edges must be unique: %s' % repr(bins))
+        if (duplicate_edges == 'raise'):
+            raise ValueError('Bin edges must be unique: %s'
+                             % repr(bins))
```

Comment:
expand this message to say, you can force edges to be unique by passing ``errors='drop'``